Presentation.Web/app/components/org/user/org-user.controller.ts

```diff
@@ -418,6 +435,10 @@
 
             this.mainGridOptions = mainGridOptions;
         }
+        
+        private exportToExcel = (e: IKendoGridExcelExportEvent<Models.ItSystem.IItSystem>) => {
```

Comment:
"Kun viste kolonner" does not work... so something is offf in this one.. Works fine in IT-System catalog using the same integration... 

![image](https://user-images.githubusercontent.com/13116889/206106676-fde5cd7d-cfcf-42ee-8c3c-6fd31110bf4b.png)